src/discord.ts

```diff
@@ -86,26 +88,23 @@ discord.on(
   },
 );
 
-export const sendLiveAnnouncement = async (streamInfo: StreamInfo) => {
-  if (announcementsChannel) {
-    const user = await UserManager.getUserById(streamInfo.user_id);
-    const started_at = new Date(streamInfo.started_at);
-
-    // Fetch category name
-    let category = await fetchGameById(streamInfo.game_id);
-    if (!category) {
-      category = { name: "" };
-    }
+export const sendTwitchLiveAnnouncement = async (streamInfo: StreamInfo) => {
```

Comment:
Why do we have a `sendTwitchLiveAnnouncement` but no `sendGlimeshLiveAnnouncement`? Not sure this method makes sense, or if the caller should do this, especially since that's what you're doing for Glimesh

Reply:
My two cents here is to have a `sendLiveAnnoucement(streamInfo: StreamInfo, service: Service)` where `Service` is an enum of `Twitch`, `Glimesh` etc that allows easier expansion in the future. For example, if we wanted to add YouTube or Facebook Gaming (please no).

Reply:
Good thought, but probably better for the caller to know which to send, since that's what's interfacing with the service. The Discord class should only care about nuances with the Discord interface, and expect the Twitch and Glimesh classes to care about coercing their data into the 'common' format.